libnmstate/varlink/README.varlink.md

```diff
@@ -0,0 +1,125 @@
+# Varlink support for libnmstate
+Varlink protocol ([Varlink.org](https://varlink.org/)) can used to communicate with libnmstate via stdin/out and varlink client implementation ([Existing Language binding](https://varlink.org/Language-Bindings#how-to-test-new-language-bindings)). Varlink protocol encodes all messages as JSON object and communicates over unix and tcp socket connections. Nmstate service is defined in io.nmstate interface address and libnmstate functions should called in as specified in varlink interface defination file `io.nmstate.varlink`.
+
+## Initiate nmstate varlink service
+unix socket connection
+```
+$ io.nmstate unix:@nmstate &
+```
+tcp socket connection
+```
+$ io.nmstate tcp:127.0.0.1:8000
+```
+systemd service initiation for unix and tcp
```

Comment:
This PR does not contains systemd unit files.

Reply:
Systemd service implemented in #1193. I'll updated it other PR also.